### ISSUE TITLE
missing password.

### ISSUE THREAD

**Issue by molinf** [dalaranwow/dalaran-wow#4858]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: im missing my old password for my old account. 
**Expected behaviour**: get in contact with gm. 
**Steps to reproduce the problem**:

1. have contact with me. 
2. reset password to connected email
3. happy customer.

**Include proofs for this behaviour**
i have made an ticket in game, sent email to your dalaran.community@gmail.com but hasnt recvieved any service what so ever. this was done in early june.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
